src/legacy/core_plugins/console/np_ready/public/application/containers/editor/legacy/console_editor/console_editor.tsx

```diff
@@ -17,14 +17,17 @@
  * under the License.
  */
```

Comment:
TL;DR don't mind too much either way personally, but happy to make the change since it's a bit less code that way :)

Hmm, I wrote this with something like this in view: https://github.com/elastic/kibana/pull/41790/files#diff-186385d9ca4564b5260ba2fec9c7057dR1

Where we would, internally, be using the same component. This was an attempt to mimic that structure. That being said, even when we do wind up using the underlying editor component we will probably still wrap it in our own components.